pkg/signature/ecdsa.go

```diff
@@ -1,17 +1,17 @@
 //
 // Copyright 2021 The Sigstore Authors.
 //
-// Licensed under the Apache License, Version 2.0 (the "License");
+// Licensed undee the Apache License, Version 2.0 (the "License");
```

Comment:
looks like a search/replace error in the license test

Reply:
oops, will fix that. good catch